pkg/spiffs/fs/spiffs_fs.c

```diff
@@ -340,6 +340,29 @@ static int _fsync(vfs_file_t *filp)
     return spiffs_err_to_errno(ret);
 }
 
+static void spiffs_stat_to_stat(const spiffs_stat *in, struct stat *out)
+{
+    /* stat.name; */
+    out->st_ino = in->obj_id;
+    /* stat.pix; */
+    out->st_size = in->size;
+    /* stat.type;*/
+    out->st_mode = S_IFREG;
```

Comment:
spiffs does not support directories, so there are only files

Reply:
That has been around since SPIFFS' start, looks more like "we better define a few codes so we don't block our way" or "hey maybe we ever want, so we better".

Reply:
There is also a new version of spiffs in the making for several years now (development stopped for quite some time, but I see now there are new commits again).

This will be incompatible to existing SPIFFS filesystems, so it will be a similar situation like with littlefs -> littlefs2